webserver/static/scripts/datasets/eval-jobs-viewer.js

```diff
@@ -1,3 +1,4 @@
+/** @jsx React.DOM */
```

Comment:
https://facebook.github.io/react/blog/2014/10/16/react-v0.12-rc1.html#the-jsx-pragma-is-gone